scanner/src/main/kotlin/Scanner.kt

```diff
@@ -777,8 +771,7 @@ fun ScanResult.toNestedProvenanceScanResult(nestedProvenance: NestedProvenance):
 
     val provenances = nestedProvenance.getProvenances()
     val scanResultsByProvenance = provenances.associateWith { provenance ->
-        // TODO: Find a solution for the incorrect packageVerificationCode and for how to associate issues to the
-        //       correct scan result.
+        // TODO: Find a solution for how to associate issues to the correct scan result.
```

Comment:
Isn't the change of this comment actually a fixup for calculating the PVC on the fly?

Reply:
BTW, while double-checking I also discovered the comment at https://github.com/oss-review-toolkit/ort/blob/70a13d2d70c7c54ddcc446b74580e459c01883ff/model/src/main/kotlin/utils/ScanResultUtils.kt#L37-L38 which could also be adjusted now.

Reply:
And also https://github.com/oss-review-toolkit/ort/blob/70a13d2d70c7c54ddcc446b74580e459c01883ff/scanner/src/main/kotlin/provenance/NestedProvenanceScanResult.kt#L67-L68

Reply:
> Isn't the change of this comment actually a fixup for calculating the PVC on the fly?

Maybe it depends on the point of view. If one say the scanner does not have any knowledge about any "reporters", which I believe is true, then probably not. Also my approach was: 1. Implement a replacement for the non working existing package verification code first (previous PR) 2. Then remove all the left-over in the scanner (this PR). That separation still makes sense to me, so I'd say it's not a fix-up.